pkgs/development/python-modules/vapoursynth/default.nix

```
@@ -0,0 +1,23 @@
+{ buildPythonPackage, cython, lib, vapoursynth }:
+buildPythonPackage {
+  pname = "vapoursynth";
+  inherit (vapoursynth) src version;
+
+  nativeBuildInputs = [
+    cython
+  ];
+
+  buildInputs = [
+    vapoursynth
+  ];
+
+  checkPhase = ''
+    python -m unittest discover -s $src/test -p "*test.py"
+  '';
+
+  meta = with lib; {
+    inherit (vapoursynth.meta) description homepage license platforms;
+    maintainers = [];
+  };
```

```suggestion
  inherit (vapoursynth) meta;
```
We shouldn't clear maintainers. Also we want to inherit everything in case things get added or changed.